MainModule/Server/Commands/Fun.lua

```diff
@@ -4773,8 +4773,7 @@ return function(Vargs, env)
 			Fun = true;
 			AdminLevel = "Admins";
 			Function = function(plr: Player, args: {string})
-				local num = assert(tonumber(args[1]), "Missing gravity value (or enter 'fix' to reset to normal)'")
-				workspace.Gravity = num or 196.2
+				workspace.Gravity = args[1] == "fix" and 196.2 or assert(tonumber(args[1]), "Missing gravity value (or enter 'fix' to reset to normal)'")
```

Comment:
I suppose a change to "default" would be more suitable as _some_ games do change their default `workspace.Gravity` property or allow the "fix" argument to fetch the old Gravity and actually "fix" it (i.e: caching old  before changing gravity, etc)

Reply:
Done